### PR TITLE
Remove unused from _libs.parsers

### PR DESCRIPTION
**Pull request by jbrockmendel**
- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: pandas/_libs/parsers.pyx [hunk 1]
```diff
@@ -1479,6 +1449,7 @@ cdef _string_box_factorize(parser_t *parser, int64_t col,
 
     return result, na_count
 
+
```
Comment (jreback) [inline pandas/_libs/parsers.pyx]: are we not linting these .pyx for the 2 blank lines between functions?

**Review by jreback — APPROVED** [review]
lgtm. just a question. ping on green.

File: pandas/_libs/parsers.pyx (hunk 1 above)
Comment (jbrockmendel) [reply pandas/_libs/parsers.pyx]: flake8 chokes on a SyntaxError as soon as it hits a `cimport`

**Comment by jbrockmendel** [comment]
ping

**Comment by jreback** [comment]
thanks!

**Pull request merged**